Disable apps

bengillies opened the pull request:
This temporarily disables the app switcher (note that it is still accessible from the /apps uri) so that it is no longer the default page that loads up on new spaces.

bengillies committed:
Temporarily disable the app switcher as default

This changes the default page for new spaces back to TiddlyWiki. This is
because the app switcher doesn't really give any direction or guidance
as to what users are supposed to do. In short, it's not ready yet.

This commit just comments out the code in the function (rather than
removing it) on the basis that the app switcher is a better long term
home page, but isn't the best one at the moment.

bengillies committed:
Disabled app-switcher-as-default test

This will need to be re-enabled when the app switcher is put back as the
default page for new spaces

cdent commented:
I reckon this is a good idea but it would be even better if we could make yet another revision to GettingStarted (`src/system-info/GettingStarted.tid`) to make the introduction a bit better. Even if just a link to a page somewhere "Your first time on TiddlySpace".

Something to smooth out the transition even more.

Any chance of getting that in too?

bengillies commented:
Yep, that makes sense. I'll add something tomorrow.

bengillies committed:
Changed the GettingStarted text

It should now tell a much more coherent story, which should gradually
unveil more features while directing the new user to upload an icon,
create a tiddler, include a space, etc.

It's based on Jon's caspian-ii space, though obviously tweaked a bit to
fit in with the default theme.

bengillies commented:
I reckon this is good for pulling now.
